static-site/content/SARS-CoV-2-Datasets.yaml

```diff
@@ -728,6 +728,17 @@ datasets:
     org:
       name: CITRIC Center for Bioinformatics Aga Khan University Pakistan
       url: https://www.aku.edu/Pages/home.aspx
+      
+  - url: https://nextstrain.org/community/AKU-CITRIC-Center-for-Bioinformatics/ncov/Pakistan_Contextual
+    name: Pakistan
+    geo: pakistan
+    level: country
+    coords:
+    - 68.3451
+    - 29
```

Comment:
This (longitude, latitude) pair is the same as your other Pakistan dataset, so the two dots would overlap on the map. Before merging, I'd plan to update the longitude to be +1 to space it out. This is how your other dataset is separated out on the map from the NIH BIGVI dataset, for example.